File: php/src/vaas/Message/Kind.php
<?php

namespace VaasSdk\Message;

enum Kind: string
{
    case AuthRequest = "AuthRequest";
    case AuthResponse = "AuthResponse";
    case VerdictRequest = "VerdictRequest";
    case VerdictResponse = "VerdictResponse";
    case VerdictRequestForUrl = "VerdictRequestForUrl";
    case VerdictRequestForStream = "VerdictRequestForStream";
    case Error = "Error";
}

File: java/src/main/java/de/gdata/vaas/messages/Kind.java
package de.gdata.vaas.messages;

import com.google.gson.annotations.SerializedName;

public enum Kind {
    @SerializedName("VerdictRequest")
    VerdictRequest,
    @SerializedName("VerdictResponse")
    VerdictResponse,
    @SerializedName("AuthRequest")
    AuthRequest,
    @SerializedName("AuthResponse")
    AuthResponse,
    @SerializedName("Error")
    Error,
    @SerializedName("VerdictRequestForUrl")
    VerdictRequestForUrl
}    
